docs/devices/TS0201.md

```diff
@@ -34,6 +34,7 @@ After this the device will automatically join.
 
 ## OTA updates
 This device supports OTA updates, for more information see [OTA updates](../guide/usage/ota_updates.md).
+For the OTA update to work on this device, you need to keep pressing the button on the device again and again until the update is complete so the device doesn´t lose the connection with zigbee2mqtt.
```

Comment:
Please move above: `<!-- Notes END: Do not edit below this line -->`

Reply:
Done, and I have included a note so other people are aware of the huge issue with the latest firmware that breaks this device.